DO NOT USE - Internal validation review hn (#5)
mulesoft/docs-hosting: modules/ROOT/pages/ch2-private-space-about.adoc

You can create multiple private spaces, either in the same or different regions.

You connect your private intranet to your private space to function as a single, private network. 

In each private space, you define:

* A _private network_, which is a virtual cloud where apps deployed to this private space run.
* One or more connections from the private network to your external network, either via Anypoint VPN or a transit gateway.
* TLS contexts, which define the domains that are available when deploying apps to the private spaces, and optionally enable mutual TLS.
* Firewall rules to allow and block inbound and outbound traffic to your private space.
* The environments and business groups to allow to deploy to the private space.
// * Where to forward logs, either Anypoint Monitoring or an external service such as Splunk.

// Utilize Anypoint Security Edge policies

== Private Network

When you create the private network, you associate a range of IP addresses for the apps in your private space to use, the region in which they run, and optionally, any internal DNS servers to resolve requests to custom domains.

> Private spaces support Amazon regions.

Reviewer: What is this supposed to mean?

and optionally enable mutual TLS. 

When deploying an app to the private space, you can use the domains from the TLS contexts
to configure multiple public endpoints, which clients use to reach the app from the internet.
If the domains in the TLS context include a wildcard, you can configure an optional subdomain, 
such as the app name or an organization.
You can create as many endpoints as you need.

== Path Rewriting

You can configure the app to be accessible from
a different path from the base path where the app is listening.

== Firewall Rules

You control the traffic to and from your private spaces.

Inbound::
By default, CloudHub 2.0 blocks all traffic to the private space and provides firewall rules to allow connections through specified ports. 
You can configure additional firewall rules to control traffic to your private space. 
+
The private space provides load balancing to balance inbound traffic to apps across multiple replicas.
Outbound::
By default, CloudHub 2.0 allows HTTPS traffic on port 443 from the private space.
You can configure additional firewall rules to configure additional HTTPS and HTTP connections on port 80.

> By combining firewall rules and TLS context configuration, you can fine-tune how external services reach your applications.

Reviewer: ```suggestion
By combining firewall rules and TLS context configuration, you can fine-tune the ingress and egress of your application traffic.
```
